omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1606 @@
+^^^^^^^^^^
+
+`General overview <http://www.postgresql.org/support/versioning/>`__
+
+.. tabularcolumns:: |l|l|l|l|l|
+
+.. list-table::
+    :header-rows: 1
+
+    * - Version
+      - Release date
+      - Upstream support
+      - OME (build)
+      - OME (deploy)
+    * - 8.4
+      - from Jul 2009
+      - to Jul 2014
+      - Supported
+      - Supported
+    * - 9.0
+      - from Sep 2010
+      - to Sep 2015
+      - Supported
+      - Supported
+    * - 9.1
+      - from Sep 2011
+      - to Sep 2016
+      - Supported
+      - Supported
+    * - 9.2
+      - from Sep 2012
+      - to Sep 2017
+      - Recommended
+      - Recommended
+    * - 9.3
+      - from Sep 2013
+      - to Sep 2018
+      - Recommended
+      - Recommended
+    * - 9.4
+      - from Dec 2014
+      - to Dec 2019
+      - Upcoming
+      - Upcoming
+    * - 9.5
+      - TBA
+      - TBA
+      - Upcoming
+      - Upcoming
+    * - Details
+      - 
+      - `Link <http://www.postgresql.org/support/versioning/>`__
+      - 
+      - 
+
+Distribution support:
+
+.. tabularcolumns:: |l|l|L|L|L|l|l|l|
+
+.. list-table::
+    :header-rows: 1
+
+    * - Version
+      - CentOS/RHEL
+      - Fedora
+      - Ubuntu
+      - Debian
+      - Homebrew
+      - FreeBSD Ports
+    * - 8.4
+      - 6.x
+      - N/A
+      - 10.04, 12.04
+      - 6.0, 7.0
+      - Yes
+      - Yes
+    * - 9.0
+      - N/A
+      - N/A
+      - N/A
+      - N/A
+      - Yes
+      - Yes
+    * - 9.1
+      - N/A
+      - N/A
+      - 12.04, 13.04, 13.10
+      - 7.0, (8.0), 6.0 bpo
+      - Yes
+      - Yes
+    * - 9.2
+      - 7.x
+      - 19
+      - N/A
+      - N/A
+      - Yes
+      - Yes
+    * - 9.3
+      - N/A
+      - 20, 21
+      - 14.04
+      - N/A
+      - N/A
+      - Yes
+    * - 9.4
+      - N/A
+      - N/A
+      - 14.10
+      - (8.0)
+      - N/A
+      - Yes
+    * - 9.5
+      - N/A
+      - N/A
+      - N/A
+      - N/A
+      - N/A
+      - N/A
+    * - Details
+      - 
+      - `Link <https://apps.fedoraproject.org/packages/postgresql>`__
+      - `Link <http://packages.ubuntu.com/search?keywords=postgresql&searchon=names&suite=all&section=all>`__
+      - `Link <https://packages.debian.org/search?keywords=postgresql&searchon=sourcenames&suite=all&section=all>`__
+      - 
+      - 
+
+The PostgreSQL project provides `packages
+<http://www.postgresql.org/download/>`__ for supported platform.
+Therefore distribution support is not critical since 9.3 and 9.4 are
+available for all platforms.
+
+Python
+^^^^^^
+
+.. tabularcolumns:: |l|l|l|l|l|l|
+
+.. list-table::
+    :header-rows: 1
+
+    * - Version
+      - Release date
+      - Upstream support
+      - OME (build)
+      - OME (deploy)
+      - Details
+    * - 2.5
+      - from Sep 2006
+      - to Oct 2011
+      - Dropped
+      - Dropped
+      - `Link 1 <https://mail.python.org/pipermail/python-committers/2011-October/001844.html>`__
+        `Link 2 <https://www.python.org/download/releases/2.5/>`__
```

Comment:
The only real purpose of these links is to document where the information came from and provide a hint for where to update it for new releases.  We can certainly make it nicer than "Link", but extended descriptions will screw up the table layouts.